content/redirect/cli-command-requires-channel.adoc

```diff
@@ -0,0 +1,6 @@
+---
+layout: redirect
+redirect_url: https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+CLI
```

Comment:
Tell me where and I'll change it. For now I just want _something_ here, as the site can trivially be changed, but we cannot replace URLs in the Jenkins versions people use.